src/System.Memory/tests/Performance/Perf.ReadOnlySequence.Enumerator.cs

```diff
@@ -0,0 +1,301 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Memory.Tests;
+using System.MemoryTests;
+using Microsoft.Xunit.Performance;
+using Xunit;
+
+namespace System.Buffers.Tests
+{
+    public class Perf_ReadOnlySequence_Enumerator
+    {
+        private const int InnerCount = 10_000;
+        volatile static int _volatileInt = 0;
+
+        [Benchmark(InnerIterationCount = InnerCount)]
+        [InlineData(10_000, 100)]
+        private static void Byte_Array(int bufSize, int bufOffset)
+        {
+            var buffer = new ReadOnlySequence<byte>(new byte[bufSize], bufOffset, bufSize - 2 * bufOffset);
+
+            foreach (BenchmarkIteration iteration in Benchmark.Iterations)
+            {
+                int localInt = 0;
+                using (iteration.StartMeasurement())
+                {
+                    for (int i = 0; i < Benchmark.InnerIterationCount * 10; i++)
+                    {
+                        foreach(ReadOnlyMemory<byte> memory in buffer)
+                            localInt ^= memory.Length;
+                    }
+                }
+                _volatileInt = localInt;
+            }
+        }
+
+        [Benchmark(InnerIterationCount = InnerCount)]
```

Comment:
Instead of doing ` for (int i = 0; i < Benchmark.InnerIterationCount * 10; i++)`, change the InnerIterationCount here: `[Benchmark(InnerIterationCount = InnerCount * 10)]`

Same in other places.

Reply:
I made it for equal number of iteration between this test and multi segment tests.

Reply:
Why? It is cleaner to change inner iteration count within the attribute rather than changing it within the loop.

Reply:
Thank I will change it in attribute.